Test/AwsCommonRuntimeKitTests/XCBaseTestCase.swift

```diff
@@ -69,6 +69,12 @@ extension XCTestCase {
     #endif
   }
 
+  func skipIfLocalhostUnavailable() throws {
+    guard let _ = ProcessInfo.processInfo.environment["aws.crt.localhost"] else {
```

Comment:
let's add comments about how to enable the local host, and how it varies from the localhost used from other places (eg: we have MQTT client uses localhost, and proxies.)

I think it may worth a section in the readme about how to run those tests. And how to set them up.